woodwork/logical_types.py

```diff
@@ -503,3 +503,15 @@ class PostalCode(LogicalType):
     primary_dtype = 'category'
     backup_dtype = 'string'
     standard_tags = {'category'}
+
+
+_NULLABLE_PHYSICAL_TYPES = {
+    'Int64',
```

Comment:
You can add more nullable types:

```suggestion
    'Int8',
    'Int16',
    'Int32',
    'Int64',
    'Float8'
    'Float16'
    'Float32'
    'Float64'
```